units/XNL0106/XNL0106_Script.lua

```diff
@@ -1,13 +1,12 @@
--- T1 light tank (should match LABs in power)
-
 local NHoverLandUnit = import('/lua/nomadsunits.lua').NHoverLandUnit
 local DarkMatterWeapon1 = import('/lua/nomadsweapons.lua').DarkMatterWeapon1
-local SlowHover = import('/lua/defaultunits.lua').SlowHoverLandUnit
+local SlowHoverLandUnit = import('/lua/defaultunits.lua').SlowHoverLandUnit
 
-XNL0106 = Class(NHoverLandUnit, SlowHover) {
+--- Tech 1 Light Tank
```

Comment:
May be confusing since normally all tanks aren't as weak as LABs.
```suggestion
--- Tech 1 Light Tank (LAB Equivalent)
```